src/test/java/bio/terra/app/controller/AssetModelValidationTest.java

```diff
@@ -0,0 +1,106 @@
+package bio.terra.app.controller;
```

Comment:
I would suggest that (once you review my PR and fault insertion is in), that you add a fault that forces an exception in the doStep of create to make sure the undo works. And then use it in a test here.